samples/single_tenant_app/app.py

```diff
@@ -0,0 +1,136 @@
+import json
+from typing import Dict, List
+from fastapi import Depends, Security
+from fastapi.responses import PlainTextResponse
+
+from pydantic import BaseSettings
+from saleor_app.deps import ConfigurationDataDeps, ConfigurationFormDeps
+from saleor_app.schemas.core import Saleor, WebhookData
+from saleor_app.app import SaleorApp
+from saleor_app.saleor.utils import get_client_for_app
+from saleor_app.schemas.handlers import SaleorEventType
+from saleor_app.schemas.webhook import Webhook
+from saleor_app.security import SaleorWebhookVerification, saleor_webhook_security
+from starlette.middleware.cors import CORSMiddleware
+
+from single_tenant_app.db import (
+    create_tables,
+    upsert_app_data,
+    retrieve_app_data,
+    database,
+)
+from single_tenant_app.manifest import manifest
+
+
+class Settings(BaseSettings):
+    debug: bool = False
+    saleor_domain: str
+    use_insecure_saleor_http: bool = False
+
+
+settings = Settings(
+    debug=True,
+    saleor_domain="172.17.0.1:8000",
```

Comment:
If this is a sample app for the wider audience I would add a comment here on why this IP is here and how to handle it for mac users.